InvenTree/plugin/builtin/barcodes/digikey_barcode.py

```diff
@@ -0,0 +1,66 @@
+"""The DigiKeyBarcodePlugin matches DigiKey barcodes to supplier parts."""
+
+import logging
+
+from django.utils.translation import gettext_lazy as _
+
+from company.models import SupplierPart
+from plugin import InvenTreePlugin
+from plugin.mixins import BarcodeMixin
+
+logger = logging.getLogger('inventree')
+
+
+class DigiKeyBarcodePlugin(BarcodeMixin, InvenTreePlugin):
+    """BarcodePlugin for matching DigiKey barcodes."""
+
+    NAME = "DigiKeyBarcode"
+    TITLE = _("DigiKey Barcodes")
+    DESCRIPTION = _("Provides support for scanning DigiKey barcodes")
+    VERSION = "0.1.0"
```

Comment:
Suggestion: If it works it should be 1.0

Reply:
Done.